Makefile

```diff
@@ -1,10 +1,11 @@
 EXAMPLES := $(shell ./get_main_pkgs.sh ./example)
+TOOLS_MOD_DIR := ./tools
 
 # All source code and documents. Used in spell check.
 ALL_DOCS := $(shell find . -name '*.md' -type f | sort)
 # All directories with go.mod files. Used in go mod tidy.
```

Comment:
While following what @rghetia wrote in the review, could you update this comment? Maybe something like:
```
# All directories with go.mod files related to opentelemetry library. Used for building, testing and linting.
```